### PR TITLE
fix: only lint PR title format

### LINKED ISSUE #856: Make Conventional Commits syntax optional for contributors
Discussed [here](https://github.com/honeybadger-io/honeybadger-js/pull/844#pullrequestreview-1068961472).

TODO:
- [ ] Update readme page to say that CC syntax is optional
- [ ] Research on how to enforce CC on main branch

### PR DESCRIPTION
**Pull request by shalvah**
Closes #856

### CODE REVIEW

**Review by subzero10 — APPROVED** [review]
This looks good to me!
**FYI, I went ahead and set the default commit message to always be the PR title.** We have to do this because [single commit PRs do not use the PR title when squashing](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#merge-message-for-a-squash-merge).

cc @joshuap 
Screenshot 2022-10-14 at 1 03 20 PM

File: .github/workflows/commitlint.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14.x'
+          cache: 'npm'
+          cache-dependency-path: |
+            package-lock.json
+            packages/js/examples/aws-lambda-typescript/package-lock.json
```
[the remark this answers is not in the document]
Comment (shalvah) [reply .github/workflows/commitlint.yml:22]: 👍

**Comment by shalvah** [comment]
Oh, thanks. Strange choice by GitHub.

**Review by joshuap — DISMISSED** [review]
🚀

**Review by subzero10 — APPROVED** [review]

**Review by subzero10 — APPROVED** [review]

**Pull request merged**